cores/arduino/Adafruit_TinyUSB_Core/tinyusb/src/class/cdc/cdc_device.h

```diff
@@ -168,11 +167,6 @@ static inline bool tud_cdc_peek (int pos, uint8_t* u8)
   return tud_cdc_n_peek(0, pos, u8);
 }
 
-static inline uint32_t tud_cdc_write_char (char ch)

```

Comment:
please keep this function, it is part of the API of the tinyusb stack, it is OK to be unused

Reply:
remove commit that change .h so this should sadisfy your request